Corona-Warn-App/src/main/res/layout/home_statistics_cards_local_incidence_layout.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/incidence_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:ignore="MissingConstraints"
```

Comment:
Why are we ignoring this?
The warning usually means that in some layout/language setups, the views may jump to a specific direction due to missing constraints.

Reply:
no idea tbh, i stole it from home_statistics_cards_incidence_layout.xml

Reply:
turns out this does nothing, the ide does not complain if i remove it 😄